simulator/simple_planning_simulator/include/simple_planning_simulator/vehicle_model/sim_model_delay_steer_map_acc_geared.hpp

```diff
@@ -0,0 +1,215 @@
+// Copyright 2021 The Autoware Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SIMPLE_PLANNING_SIMULATOR__VEHICLE_MODEL__SIM_MODEL_DELAY_STEER_MAP_ACC_GEARED_HPP_
+#define SIMPLE_PLANNING_SIMULATOR__VEHICLE_MODEL__SIM_MODEL_DELAY_STEER_MAP_ACC_GEARED_HPP_
+
+#include "eigen3/Eigen/Core"
+#include "eigen3/Eigen/LU"
+#include "interpolation/linear_interpolation.hpp"
+#include "raw_vehicle_cmd_converter/csv_loader.hpp"
+#include "simple_planning_simulator/vehicle_model/sim_model_interface.hpp"
+
+#include <common/types.hpp>
+
+#include <deque>
+#include <iostream>
+#include <queue>
+#include <string>
+#include <vector>
+
+using float64_t = autoware::common::types::float64_t;
```

Comment:
Please do not use the autoware-defined types. Replace it with double as other models do.

Reply:
fixed in eeca004f90238d36e84304b9d05ec0fbee6e97fa